src/styles/blocks/card.scss

```diff
@@ -0,0 +1,95 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+
+  &__image {
+    display: block;
+    width: 160px;
```

Comment:
The card image width is set to 160px. It should be set to 166px to match the design spec and maintain consistency with the card content width.